tests/matrix_free/matrix_vector_rt_face_common.h

```diff
@@ -225,17 +225,21 @@ do_test(const DoFHandler<dim> &          dof,
 {
   deallog << "Testing " << enum_to_string(test_type) << std::endl;
 
+  constexpr unsigned int n_q_points = fe_degree + 2;
+  constexpr unsigned int m_degree   = fe_degree + 2;
+
+  const MappingQ<dim>     mapping(m_degree);
   MatrixFree<dim, Number> mf_data;
   {
-    const QGaussLobatto<1>                           quad(fe_degree + 2);
-    const MappingQ<dim>                              mapping(fe_degree);
+    const QGaussLobatto<1>                           quad(n_q_points);
     typename MatrixFree<dim, Number>::AdditionalData data;
     data.tasks_parallel_scheme = MatrixFree<dim, Number>::AdditionalData::none;
-    data.mapping_update_flags  = update_gradients | update_JxW_values;
+    data.mapping_update_flags =
+      (update_gradients | update_JxW_values | update_hessians);
     data.mapping_update_flags_inner_faces =
-      (update_gradients | update_JxW_values);
+      (update_gradients | update_JxW_values | update_hessians);
```

Comment:
Looking at this, I guess we should make sure that we translate that information also from `update_piola`, because that is the natural thing to request for RT elements. But that does not need to happen in this PR.